src/Algebra/Structures.agda

```diff
@@ -471,6 +428,7 @@ record IsRing (+ * : Op₂ A) (-_ : Op₁ A) (0# 1# : A) : Set (a ⊔ ℓ) where
     +-isAbelianGroup : IsAbelianGroup + 0# -_
     *-isMonoid       : IsMonoid * 1#
     distrib          : * DistributesOver +
+    zero             : Zero 0# *
```

Comment:
I'm a little bit worried about this? We're adding an entirely new axiom to `IsRing`?

Reply:
In the presence of additive inverses, this is a redundant axiom. See the deleted code below, and the definition `IsRingWithoutZero` in `Algebra.Structures.Biased`.